ui/components/multichain/network-list-menu/network-list-menu.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import Popover from '../../ui/popover/popover.component';
+import { NetworkListItem } from '../network-list-item';
+import {
+  setActiveNetwork,
+  showModal,
+  setShowTestNetworks,
+  setProviderType,
+} from '../../../store/actions';
+import { CHAIN_IDS, TEST_CHAINS } from '../../../../shared/constants/network';
+import {
+  getShowTestNetworks,
+  getAllNetworks,
+  getCurrentChainId,
+} from '../../../selectors';
+import Box from '../../ui/box/box';
+import ToggleButton from '../../ui/toggle-button';
+import {
+  DISPLAY,
+  JustifyContent,
+} from '../../../helpers/constants/design-system';
+import { Button, BUTTON_TYPES, Text } from '../../component-library';
+import { ADD_POPULAR_CUSTOM_NETWORK } from '../../../helpers/constants/routes';
+import { getEnvironmentType } from '../../../../app/scripts/lib/util';
+import { ENVIRONMENT_TYPE_FULLSCREEN } from '../../../../shared/constants/app';
+
+const UNREMOVABLE_CHAIN_IDS = [CHAIN_IDS.MAINNET, ...TEST_CHAINS];
+
+export const NetworkListMenu = ({ closeMenu }) => {
+  const t = useI18nContext();
+  const networks = useSelector(getAllNetworks);
+  const showTestNetworks = useSelector(getShowTestNetworks);
+  const currentChainId = useSelector(getCurrentChainId);
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  const environmentType = getEnvironmentType();
+  const isFullScreen = environmentType === ENVIRONMENT_TYPE_FULLSCREEN;
+
+  return (
+    <Popover onClose={closeMenu} centerTitle title={t('networkMenuHeading')}>
+      <Box>
+        <Box className="multichain-network-list-menu">
+          {networks.map((network) => {
+            const isCurrentNetwork = currentChainId === network.chainId;
+            const canDeleteNetwork =
+              !isCurrentNetwork &&
+              !UNREMOVABLE_CHAIN_IDS.includes(network.chainId);
+
+            return (
+              <NetworkListItem
+                name={network.nickname}
+                iconSrc={network?.rpcPrefs?.imageUrl || null}
```

Comment:
Since, this is using AvatarNetwork internally in NetworkListItem, we have a fallback
```suggestion
                iconSrc={network?.rpcPrefs?.imageUrl }
```